doc/Installation/Installation-(RHEL-CentOS).md

```diff
@@ -193,24 +249,53 @@ To prepare the web interface (and adding devices shortly), you'll need to create
 
 First, create and chown the `rrd` directory and create the `logs` directory
 
+```bash
     mkdir rrd logs
     chown -R librenms:librenms /opt/librenms
+    chmod 775 rrd
+
     # For HTTPd (Apache):
     chown apache:apache logs
+
     # For Nginx:
     chown nginx:nginx logs
-
-    chmod 775 rrd
+```
 
 > If you're planing on running rrdcached, make sure that the path is also chmod'ed to 775 and chown'ed to librenms:librenms.
 
+**SELinux**
+> if you're using SELinux you need to allow nginx to write into logs directory.
```

Comment:
It's not only `nginx` that needs to be able to write into logs :)

Could we make it more bold that we dont currently support SELinux at all and that we're still figuring a correct policy?